cmd/translatesfx/translatesfx/testdata/otel-e2e-simple-expected.yaml

```diff
@@ -40,8 +43,16 @@ service:
     metrics:
       receivers:
         - smartagent/collectd/redis
+        - smartagent/signalfx-forwarder
       processors:
         - resourcedetection
         - metricstransform
       exporters:
         - signalfx
+    traces:
```

Comment:
nit, the order is reversed from the convention

Reply:
Traces tend to go first? Didn't know that, will fix.

Reply:
Oh, I see, you're referring to exporters being above receivers. Will fix that instead 😄